src/sqs/index.ts

```diff
@@ -318,9 +325,9 @@ export class SqsQueue<MessagePayload> {
 			executionId: uuidv4(),
 			queue: this.queueName})
 
-		const receiveCount = Number(message.MessageAttributes?.ApproximateReceiveCount?.StringValue || "1");
+		const receiveCount = Number(message.Attributes?.ApproximateReceiveCount || "1");
 
-		const context: Context<MessagePayload> = {message, payload, log, receiveCount: receiveCount}
+		const context: Context<MessagePayload> = {message, payload, log, receiveCount: receiveCount, lastAttempt: receiveCount===this.maxAttempts}
```

Comment:
nit: Shall we maybe use `>=`, just in case? :)

Reply:
Oh yes, exactly! In case of DLQ replay